ensembl_metadata/models/__init__.py

```diff
@@ -0,0 +1,4 @@
+from ensembl_metadata.models.assembly import *
```

Comment:
I'm not sure if I need to change anything here - it wasn't working with the models in subdirectories, so I [looked it up](https://miro.medium.com/max/914/1*gU01cjQAw_EOBcLaGWg7IA.jpeg) and this was the suggested fix. I think this is what Django needs to be able to find the models in the subdirectories.

Reply:
It's just listing the actual classes you want to be imported with `*`, not more, and indeed, since there is not too much transitive import in the models files, that's ok. Was mentioning it for you to know more about it :-)